iceoryx_posh/source/roudi/service_registry.cpp

```diff
@@ -26,16 +26,17 @@ ServiceRegistry::ServiceDescriptionEntry::ServiceDescriptionEntry(const capro::S
 {
 }
 
-cxx::expected<ServiceRegistry::Error> ServiceRegistry::add(const capro::ServiceDescription& serviceDescription) noexcept
+cxx::expected<ServiceRegistry::Error> ServiceRegistry::add(const capro::ServiceDescription& serviceDescription,
+                                                           ReferenceCounter_t ServiceDescriptionEntry::*count)
```

Comment:
Could you please a reference instead of a pointer.

Reply:
It is a pointer to member.